src/CodeNamerGo.cs

```diff
@@ -26,17 +26,41 @@ public new static CodeNamerGo Instance
             }
         }
 
-        public virtual IEnumerable<string> AutorestImports => new string[] { PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest") };
+        public virtual IEnumerable<string> AutorestImports
+        {
+            get
+            {
+                if (TemplateFactory.Instance.TemplateVersion != TemplateFactory.Version.v1)
+                {
+                    return new string[] { PrimaryTypeGo.GetImportLine(package: "github.com/Azure/requestpolicy-go/pipeline") };
+                }
+                return new string[] { PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest") };
+            }
+        }
 
-        public virtual IEnumerable<string> StandardImports => new string[] 
-        { 
-            PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest/azure"), 
-            PrimaryTypeGo.GetImportLine(package: "net/http") 
-        };
+        public virtual IEnumerable<string> StandardImports
+        {
+            get
+            {
+                var imports = new List<string>();
+                imports.Add(PrimaryTypeGo.GetImportLine(package: "net/http"));
+                if (TemplateFactory.Instance.TemplateVersion != TemplateFactory.Version.v1)
+                {
+                    imports.Add(PrimaryTypeGo.GetImportLine(package: "context"));
+                    imports.Add(PrimaryTypeGo.GetImportLine(package: "fmt"));
+                    imports.Add(PrimaryTypeGo.GetImportLine(package: "net/url"));
+                }
+                else
+                {
+                    imports.Add(PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest/azure"));
+                }
+                return imports;
+            }
+        }
 
         public virtual IEnumerable<string> PageableImports => new string[] 
-        { 
-            PrimaryTypeGo.GetImportLine(package: "net/http"), 
+        {
+            PrimaryTypeGo.GetImportLine(package: "net/http"),
             PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest/to") 

```

Comment:
>PrimaryTypeGo.GetImportLine(package: "github.com/Azure/go-autorest/autorest/to")  [](start = 12, length = 81)

Curiosity: will we get rid of go-autorest references? Does `requestpolicy-go` have packages for pageable?